src/app/components/nav/simple-header/simple-header.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="sim-header padding-lr-header">
```

Comment:
**Simple Header** and **Header Static** component are very similar. Can you tell why do we need both of them?

Reply:
okay, I will try to make a single component for both of them.

Reply:
Thanks! That'll be great.